crossIBC.ps1

```diff
@@ -0,0 +1,62 @@
+param(
+    [Parameter(Mandatory=$true)][string]$ToolSource,
+    [Parameter(Mandatory=$true)][string]$ToolUser,
+    [Parameter(Mandatory=$true)][string]$ToolPAT,
+    [bool]$UsePartialNgen
+)
+
+function Find-IBCMerge {
+    $IBCMergePath = gci -recurse .\.packages | where {$_.Name.Contains("ibcmerge.exe")}
+    if(!$IBCMergePath)
+    {
+        $IBCMergePath = gci -recurse C:\microsoft.dotnet.ibcmerge | where {$_.Name.Contains("ibcmerge.exe")}
+    }
+    if(!$IBCMergePath)
+    {
+        Write-Error -Category ResourceUnavailable "Could not find IBCMerge.exe"
+        exit
+    }
+    $IBCMergePath.FullName
+}
+
+function Preprocess-IBCMerge
+{
+    foreach($file in gci -recurse .packages\optimization.linux-x64.ibc.corefx\*.dll)
```

Comment:
this seems specific to Linux. how do we manage the windows ibc data?

Reply:
Windows is handled by the code in https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/codeOptimization.targets. I decided to not use a similar approach as the cross OS application was making it difficult.